webapp/src/app.tsx

```diff
@@ -2248,24 +2267,19 @@ function handleHash(hash: { cmd: string; arg: string }, loading: boolean): boole
             });
             window.location.hash = "";
             return true;
-        case "gettingstarted":
```

Comment:
There might be external links to `#gettingstarted`, and also it has a separate tick, maybe we need it?

Reply:
Yeah looking at the history, even though this does the same thing as newproject. I'll keep it.

Reply:
Done.